pulp_smash/utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging.version import Version
 from pulp_smash.constants import REPOSITORY_PATH, USER_PATH
 from time import sleep
+from urlparse import urljoin
```

Comment:
Is there a benefit to using urljoin? If so, can we switch over to using urljoin in all of the utility functions in a separate commit? (You can file a GitHub issue for this if you'd like to kick this can down the road.)